FAIMS/FAIMS3
# Conductor template admin view

# Feat/Conductor Template Admin View

## JIRA Ticket

[BSS-323](https://jira.csiro.com/browse/BSS-323)

## Description

Adds pages to the Conductor web app for managing templates. 

## Proposed Changes

New pages at /templates and /templates/:id to list current templates and allow adding, updating and deleting existing templates.

Also makes some small improvements to the Conductor page design to add a navigation bar and make it more 'normal'.

## How to Test

Run conductor, take care to initialise the db to get the new templates database.  Log in as an admin user (admin). Click on 'Templates' in the navigation bar to see the initially empty list of templates.  Click the "New Template" button to upload a notebook file as a template.  

Once you have a template you can click on it in the template list and view the page for that template.  That page allows you to upload a new version of the template or delete it. 

From the template list you can click on a template name to view the details.  From this page you can create a new notebook, update or delete the template.    Please try all of these. 

Other changes have been made:
- Changed the overall page design to use a navigation bar
- Simplified the home page, removing redundant information
- Changed the 'Copy Token' on the main page to copy just the JWT rather than the JWT + Signing key 
- On the notebook page, list notebooks that the user can modify separately - test this by creating a second user and giving them permissions on some notebooks (admin and user)

## Checklist
